### PR TITLE
Vuln addgroup

### PR DESCRIPTION
**Pull request by nastasi-oq**
Add default group and staff capability for new users.

### CODE REVIEW

File: openquakeplatform/openquakeplatform/vulnerability/models.py [hunk 1]
```diff
@@ -25,11 +25,31 @@
...
 from django.core.validators import MinValueValidator, MaxValueValidator
 from openquakeplatform.models import UnaccentCharField
-from openquakeplatform.vulnerability.widgets import TaxonomyInput, TaxonomyOutput
+from openquakeplatform.vulnerability.widgets import (TaxonomyInput,
+                                                     TaxonomyOutput)
+from django.dispatch import receiver
+from django.db.models.signals import post_save
+
+
+# somewhere, in your models.py file
```
Comment (daniviga) [inline openquakeplatform/openquakeplatform/vulnerability/models.py]: Please remove or change this comment

Comment (nastasi-oq) [inline openquakeplatform/openquakeplatform/vulnerability/models.py]: Ops! Done.

File: openquakeplatform/openquakeplatform/test/oqplatform.py [hunk 2]
```diff
@@ -105,6 +222,8 @@ def fini(self):
         #nastasi
         #<b class="caret"></b>
         #</a>
+        time.sleep(5)
+
         user_button = self.xpath_finduniq(
             "//a[@href='#' and b[@class='caret']]")
         user_button.click()
```
Comment (daniviga) [inline openquakeplatform/openquakeplatform/test/oqplatform.py]: Move this block into the `except`

**Comment by daniviga** [comment]
LGTM!

**Pull request merged**